### PR TITLE
[add-settings] Add the framework settings

### PR DESCRIPTION
**Pull request by GianlucaGuarini**

### CODE REVIEW

File: src/settings.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const SELECT_ICON = 'selectIcon';
```
Comment (nirazul) [inline src/settings.js]: I'd continue the naming scheme just like mutation types etc:
export const SELECT_ICON = ''DREIPOL_VUE_UI__SELECT_ICON';

I know, i know, it's tinfoil-hatty

File: src/settings.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+export default new Map(Object.entries({
```
Comment (nirazul) [inline src/settings.js]: Can we work around `Object.entries` for the moment? I know that support [looks good](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) but it's not that old yet in some cases (Safari, Edge) and I wouldn't rely on somebody having installed a polyfill. 
We could draw the line at ES7 stuff.

File: src/settings.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    [SELECT_ICON]: {
+        symbol: 'chevron-down',
```
Comment (nirazul) [inline src/settings.js]: Can we correct that to a sensible default that makes no assumptions about what it actually looks like? I propose `select-icon` or `select-icon-open`

File: src/components/form/README.md [hunk 4]
```diff
@@ -249,6 +249,16 @@ The select checkmark icon can be defined not only via `actions` slot but also vi
 </ui-select>
 ```
 
+You can also override the default select icon once for all via `@dreipol/vue-ui/src/settings`:
+
+```js
+import settings, {SELECT_ICON} from '@dreipol/vue-ui/src/settings';
```
Comment (nirazul) [inline src/components/form/README.md]: Can we adapt this to also use the code style of our project?
`import settings, { SELECT_ICON } from '@dreipol/vue-ui/src/settings';`

File: src/components/form/README.md [hunk 5]
```diff
@@ -249,6 +249,16 @@ The select checkmark icon can be defined not only via `actions` slot but also vi
 </ui-select>
 ```
 
+You can also override the default select icon once for all via `@dreipol/vue-ui/src/settings`:
```
Comment (nirazul) [inline src/components/form/README.md]: It's `once and for all`

File: src/settings.js (hunk 1 above)
Comment (GianlucaGuarini) [reply src/settings.js]: i have update it thanks

File: src/settings.js (hunk 2 above)
Comment (GianlucaGuarini) [reply src/settings.js]: I made this code a bit ruder but it will have a better crossbrowser support

File: src/components/form/README.md (hunk 4 above)
Comment (GianlucaGuarini) [reply src/components/form/README.md]: updated

File: src/components/form/README.md (hunk 5 above)
Comment (GianlucaGuarini) [reply src/components/form/README.md]: updated thanks

**Pull request merged**